tests/testthat/test-compute_clusters.R

```diff
@@ -0,0 +1,47 @@
+patrick::with_parameters_test_that(
+  "test compute_clusters",
+  {
+    testdata <- file.path("..", "testdata")
+
+    filenames <- sapply(files, function(x) {
+      file.path(testdata, input, paste0(x, ".parquet"))
+    })
+
+    extracted <- lapply(filenames, function(x) {
+      tibble::as_tibble(arrow::read_parquet(x)) |> dplyr::rename(rt = pos)
+    })
+
+    actual <- compute_clusters(
+      feature_tables = extracted,
+      mz_tol_relative = NA,
+      rt_tol_relative = NA,
```

Comment:
We should eventually add tests with given tolerances.

Reply:
I think there is already an issue for that #73